packages/big-design/src/components/Checkbox/Checkbox.tsx

```diff
@@ -3,14 +3,21 @@ import React, { forwardRef, useMemo, Ref } from 'react';
 
 import { useUniqueId } from '../../hooks';
 import { typedMemo, warning } from '../../utils';
+import { FormControlDescription as Description, FormControlDescriptionLinkProps } from '../Form';
```

Comment:
We should probably be consistent with the other form fields and not rename the imports.
```suggestion
import { FormControlDescription, FormControlDescriptionLinkProps } from '../Form';
```